float64 left_velocity    # Left wheel velocity
float64 right_velocity   # Right wheel velocity
float64 left_power       # Left wheel power
float64 right_power      # Right wheel power
float64 trencher_power   # Trencher motor power
float64 vibrator_power     # Vibrator motor power
float64 actuator_power   # Actuator motor power
float64 magnet_current     # Magnet current
bool is_enabled # Robot disabled state
bool is_manual # Robot control state
